doc/Type/Systemic.pod6

```diff
@@ -15,7 +15,7 @@ through the L«C<$*KERNEL>|/language/variables#Dynamic_variables» and other dyn
 
 Instance method returning the name and version of the object.
 
-    say $*PERL.gist; # OUTPUT: «Perl 6 (6.c)␤»
+    say $*PERL.gist; # OUTPUT: «Perl 6 (6.c)␤»
```

Comment:
This is code, and the output probably does not have a non-breaking space anyway.